README.md

```diff
@@ -12,7 +12,7 @@ API Blueprint is Web API documentation language. You can find API Blueprint docu
 Additionally Drafter provide set of Wrappers for serialization, of parsing result, via  [SOS library](https://github.com/apiaryio/sos) into JSON and YAML format.
 
 ## Status
-- [Format 1A7](https://github.com/apiaryio/api-blueprint/releases/tag/format-1A7) fully implemented 
+- [Format 1A8](https://github.com/apiaryio/api-blueprint/releases/tag/format-1A8) fully implemented
```

Comment:
Snowcrash is an open source library on it's own. Which is why I let it stay in it's README even though this is a bit duplicitous.